heron/config/src/yaml/conf/aurora/scheduler.yaml

```diff
@@ -8,4 +8,10 @@ heron.class.launcher:                        com.twitter.heron.scheduler.aurora.
 heron.directory.sandbox.java.home:           /usr/lib/jvm/java-1.8.0-openjdk-amd64/
 
 # Invoke the IScheduler as a library directly
-heron.scheduler.is.service:                  False
\ No newline at end of file
+heron.scheduler.is.service:                  False
+
+# Override the default Heron Aurora template
+#heron.scheduler.job.template:                heron.aurora
+
+# Override the default directory for the Aurora template
+#heron.scheduler.job.template.dir:            /etc/heron
```

Comment:
By default, this is not installed in /etc/heron. It is installed in where the cluster config resides. Location of where the config resides is provided as configPath in SubmitterMain - you need to use this configPath. Probably, a better approach would be if the config is not present, fall back to the default approach?